app/models/task.rb

```diff
@@ -17,102 +17,13 @@ class Task < ApplicationRecord
 
   self.inheritance_column = 'sti_type'
 
-  # BEGIN descriptor_to_attribute, could be move into a mixin
-
-  # TODO: move into SetDescriptorsTask
-  def get_descriptor_value(name, default = nil)
-    name_s = name.to_s
-    descriptors.each { |desc| return desc.value if desc.name.eql?(name_s) }
-    default
-  end
-
-  def set_descriptor_value(name, value, _kind = nil)
-    name_s = name.to_s
-    descriptors.each do |desc|
-      if desc.name.eql?(name_s)
-        desc.value = value
-        return
-      end
-    end
-    descriptors << Descriptor.new(name: name_s, value: value)
-  end
-
-  # END descriptors
-
-  # BEGIN subclass_to_attribute, could be move into a mixin
-  has_many :subclass_attributes, as: :attributable, dependent: :destroy, autosave: true
```

Comment:
Related to the removed class above. Only ever seemed to be used with two tasks that are no longer even in the database.